snap-geometry-edits/src/main/java/com/esri/arcgismaps/sample/snapgeometryedits/screens/MainScreen.kt

```diff
@@ -0,0 +1,192 @@
+                    mapViewInteractionOptions = MapViewInteractionOptions(isMagnifierEnabled = true),
+                    onSingleTapConfirmed = mapViewModel::identify,
+                    onPan = { mapViewModel.dismissBottomSheet() }
+                )
+                Row(
+                    modifier = Modifier
+                        .padding(12.dp)
+                        .fillMaxWidth(),
+                ) {
+                    var expanded by remember { mutableStateOf(false) }
+                    Box(
+                        modifier = Modifier
+                    ) {
+                        IconButton(
+                            enabled = mapViewModel.isCreateButtonEnabled.value,
+                            onClick = { expanded = !expanded }
+                        ) {
+                            Icon(imageVector = Icons.Default.Create, contentDescription = "Start")
+                        }
+                        DropdownMenu(
+                            expanded = expanded,
+                            onDismissRequest = { expanded = false }
+                        ) {
+                            DropdownMenuItem(
+                                text = { Text("Point") },
+                                onClick = {
+                                    mapViewModel.startEditor(GeometryType.Point)
+                                    expanded = false
+                                }
+                            )
+                            DropdownMenuItem(
+                                text = { Text("Multipoint") },
+                                onClick = {
+                                    mapViewModel.startEditor(GeometryType.Multipoint)
+                                    expanded = false
+                                }
+                            )
+                            DropdownMenuItem(
+                                text = { Text("Polyline") },
+                                onClick = {
+                                    mapViewModel.startEditor(GeometryType.Polyline)
+                                    expanded = false
+                                }
+                            )
+                            DropdownMenuItem(
+                                text = { Text("Polygon") },
+                                onClick = {
+                                    mapViewModel.startEditor(GeometryType.Polygon)
+                                    expanded = false
+                                }
+                            )
+                        }
+                    }
+                    val vector = ImageVector
+                    IconButton(
+                        enabled = mapViewModel.isUndoButtonEnabled.value,
+                        onClick = { mapViewModel.editorUndo() }
+                    ) {
+                        Icon(vector.vectorResource(R.drawable.undo), contentDescription = "Undo")
+                    }
```

Comment:
🤡  It's typical if you have an undo button that you also have a redo button.

Reply:
The design aims to have a simplified geometry editor so that we can highlight snapping. I agree that the expectation would be to have both undo and redo but at the same time question if it's necessary to have both. We can diverge from the design and add the redo if that's preferred.

Reply:
I'd think either having both, or else neither if simpler code is the ideal makes most sense to me, but having said that I'm fine to go with the concensus if only having undo was the preferred approach.